FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -529,7 +529,7 @@ public async Task<UserRecord> GetUserByPhoneNumberAsync(string phoneNumber)
         /// returned. Must have 100 entries or fewer.</param>
         /// <param name="cancellationToken">A cancellation token to monitor the asynchronous
         /// operation.</param>
-        /// <returns>A {@code Task} that resolves to the corresponding user records.</returns>
+        /// <returns>A task that resolves to the corresponding user records.</returns>
```

Comment:
DeleteUsers documentation has the same issue.

Reply:
shamecube. thanks.